src/components/js/user-family-detail.js

```diff
@@ -0,0 +1,43 @@
+import { mapActions, mapGetters } from "vuex";
+export default {
+  data() {
+    return {
+      expandedGroup: []
+    };
+  },
+  computed: {
+    ...mapGetters("user", ["userFamilies"]),
+    userId() {
+      return this.$route.params.id;
```

Comment:
We can move it to mixin aku rasa ssmua pake hahaha

Reply:
tapi isinya userId doang ?

Reply:
lupa submit bbrp review dibawah maybe bs digabungkan,
tapi kalau ndak bs dibuat utk bbrp tipical mixins juga